src/streaming/controllers/ProtectionController.js

```diff
@@ -221,10 +221,19 @@ MediaPlayer.dependencies.ProtectionController = function () {
             this.protectionModel.subscribe(MediaPlayer.models.ProtectionModel.eventList.ENAME_KEY_SESSION_REMOVED, this);
             this.protectionModel.subscribe(MediaPlayer.models.ProtectionModel.eventList.ENAME_KEY_MESSAGE, this);
 
-            // Look for ContentProtection elements.  InitData can be provided by either the
-            // dash264drm:Pssh ContentProtection format or a DRM-specific format.
-            audioInfo = aInfo;
-            videoInfo = vInfo;
+            var adapter,
+                streamInfo;
+
+            if (!aInfo || !vInfo) {
```

Comment:
@KozhinM Should this be `if (!aInfo && !vInfo)`?  There could be audio-only or video-only streams that would have one or the other, but not both.